src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -49,6 +49,10 @@
     private $filterCollection;
     private $nameConverter;
 
+    private $mimeTypes;
```

Comment:
The variables get reset on every call of the normalize function, so they are not really states.
The only other way to solve this is to pass through these variables (at least the $definitions) to nearly all other methods in the call. This makes the code much harder to read and does not make any sense at all.
If you like I can implement it that way, but I think it's better like it is now